doc/dev/Using-Mock-Test-Generation.md

```diff
@@ -0,0 +1,153 @@
+
+# Table of Contents
+1. [Generate Mock Test](#generate-mock-test)
+2. [Prepare the Mock-service-host](#prepare-mock-service-host)
+3. [Execute Mock Test](#execute-mock-test)
+4. [How to skip failure test](#skip-swagger-example)
+
+<div id="generate-mock-test"/>
+
```

Comment:
Consider to introduce more about what kind of problem we are facing and then how can mock test solve it.
Can be a link to another article.

Reply:
added in https://github.com/Azure/azure-sdk-for-net/pull/26087/commits/3865d458a76c1d7014532548fae14b357d2a2d70